.vscode/settings.json

```diff
@@ -1,5 +1,7 @@
 {
-    "python.testing.pytestArgs": [],
+    "python.testing.pytestArgs": [
+        "tests"
+    ],
```

Comment:
As above, not related to this change. I agree that it's generally useful, but it should go into python-copier-template.

Reply:
valid